plugins/extractors/bigquery/bigquery_test.go

```diff
@@ -229,6 +229,37 @@ func TestExtract(t *testing.T) {
 			utils.AssertJSONFile(t, "testdata/expected-assets.json", actual, jsondiff.FullMatch)
 		})
 	})
+
+	t.Run("with build_view_lineage true", func(t *testing.T) {
+		cfg := plugins.Config{
+			URNScope: "test-bigquery",
+			RawConfig: map[string]interface{}{
+				"project_id":             projectID,
+				"max_preview_rows":       "1",
+				"include_column_profile": "true",
+				"exclude": map[string]interface{}{
+					"datasets": []string{"exclude_this_dataset"},
+					"tables":   []string{"dataset1.exclude_this_table"},
+				},
+				"build_view_lineage": "true",
+			},
+		}
+
+		randFn := func(mainSeed int64) func(seed int64) func(max int64) int64 {
+			r := rand.New(rand.NewSource(mainSeed))
+			return func(seed int64) func(max int64) int64 {
+				return func(max int64) int64 {
+					return r.Int63n(max)
+				}
+			}
+		}
+
+		t.Run("should return upstream lineage dependencies of type view", func(t *testing.T) {
+			actual := runTest(t, cfg, randFn(1))
+
+			utils.AssertJSONFile(t, "testdata/expected-assets-mixed.json", actual, jsondiff.FullMatch)
```

Comment:
the test is failing, the testdata should be `testdata/expected-assets-view-lineage.json` right?

Reply:
it's still failing